connector_ecommerce/data/ecommerce_data.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="product_categ_services" model="product.category">
+        <field name="name">Services</field>
+    </record>
+    <record id="product_product_shipping" model="product.product">
+        <field name="default_code">SHIP</field>
+        <field name="list_price">0.0</field>
+        <field name="standard_price">0.0</field>
+        <field name="type">service</field>
+        <field name="name">Shipping Costs</field>
+        <field name="categ_id" ref="product_categ_services" />
+    </record>
+    <record id="product_product_cash_on_delivery" model="product.product">
+        <field name="default_code">CASH ON DELIVERY</field>
+        <field name="list_price">0.0</field>
+        <field name="standard_price">0.0</field>
+        <field name="type">service</field>
+        <field name="name">Cash on Delivery</field>
+        <field name="categ_id" ref="product_categ_services" />
+    </record>
+    <record id="product_product_gift" model="product.product">
+        <field name="default_code">GIFT CERTIFICATE</field>
+        <field name="list_price">0.0</field>
+        <field name="standard_price">0.0</field>
+        <field name="type">service</field>
+        <field name="name">Gift Certificate</field>
+        <field name="categ_id" ref="product_categ_services" />
+    </record>
+    <record id="product_product_discount" model="product.product">
+        <field name="default_code">DISCOUNT</field>
+        <field name="list_price">0.0</field>
+        <field name="standard_price">0.0</field>
+        <field name="type">service</field>
+        <field name="name">Discount Coupon</field>
+        <field name="categ_id" ref="product_categ_services" />
+    </record>
+    <record id="excep_parent_order_need_cancel" model="exception.rule">
+        <field name="name">Parent sales order canceled on the backend.</field>
+        <field
+            name="description"
+        >The parent sales order has been canceled on the backend. You will not be able to process this sales order until the parent's one is canceled.</field>
+        <field name="sequence">30</field>
+        <field name="model">sale.order</field>
+        <field name="code">if self.parent_need_cancel: failed = True</field>
+        <field name="active" eval="True" />
```

Comment:
I think this rule should be inactive by default as it could break standard behaviour in tests suite.
To reproduce:
- install this addon on a database
- launch a test using action_confirm (from sale_order) friom an addon not depending on connector_ecommerce
it leads to this error:
(<class 'AttributeError'>: "'sale.order' object has no attribute 'need_cancel'" while evaluating
'if self.need_cancel: failed = True')

Reply:
Thanks @Cedric-Pigeon for the comment and PR to fix. But I could not reproduce the issue. I tried testing with `sale_quotation_number` module. May I know which module did you test?